pygmt/src/plot3d.py

```diff
@@ -168,7 +168,7 @@ def plot3d(
     {p}
     {t}
         *transparency* can also be a 1d array to set varying transparency
-        for symbols.
+        for symbols, but this option is only valid if using x/y.
```

Comment:
Actually for plot3d, it should be `x/y/z`.

```suggestion
        for symbols, but this option is only valid if using x/y/z.
```